themes/theme-gmd/pages/Cart/components/Item/components/Coupon/index.jsx

```diff
@@ -70,6 +70,13 @@ class Coupon extends Component {
     this.transitionElement.style.height = `${getAbsoluteHeight(this.cardElement)}px`;
   }
 
+  /**
+   * We need to set the element height explicitly so that we can animate it later.
+   */
+  componentDidUpdate() {
```

Comment:
I think that you only need to do this once, if so then consider using `componentDidMount`

Reply:
Component props are changing. Because Coupon price (or percent value) is hidden, when coupon can not be applied (checkout is not ready, cart amount is not yet fulfilled, etc).
But once cart is fulfilled, component is updated with amount value and component height should be adjusted accordingly